<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeSeriesAnnotation</class>
 <widget class="qMRMLWidget" name="TimeSeriesAnnotation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>915</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="overlayButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="text">
        <string>Overlay (A)</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="deleteButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="text">
        <string>Delete (D)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="text">
        <string>Cycle layout</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="sliceViewButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="text">
        <string>2D/3D model</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="captureButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="text">
        <string>Capture (C)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="skipButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="text">
        <string>Skip (S)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="reviseButton">
       <property name="text">
        <string>Revise/edit past segmentations</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="text">
      <string>Segmentation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="qMRMLSegmentEditorWidget" name="segmentEditorWidget">
        <property name="defaultTerminologyEntrySettingsKey">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Inputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Input sequence browser:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="inputSequenceSelector">
        <property name="nodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>inputBrowser</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input volume:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="inputSelector">
        <property name="toolTip">
         <string>Pick the input to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>inputVolume</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Segmentation: </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="qMRMLNodeComboBox" name="segmentationSelector">
        <property name="nodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>segmentation</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Skip frames: </string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="skipFramesSpinBox">
        <property name="SlicerParameterName" stdset="0">
         <string>inputSkipNumber</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Ultrasound model: </string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="qMRMLNodeComboBox" name="ultrasoundModelSelector">
        <property name="nodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>ultrasoundModel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Segmentation browser: </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="segmentationBrowserSelector">
        <property name="nodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>segmentationBrowser</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton">
     <property name="text">
      <string>Outputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Filename prefix: </string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="exportSequenceButton">
        <property name="text">
         <string>Export sequence to files</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="reconstructSegmentationsButton">
        <property name="text">
         <string>Reconstruct segmentations to volume</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Output folder: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ctkPathLineEdit" name="outputPathLineEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="filenamePrefixLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="sampleDataButton">
     <property name="text">
      <string>Load sample data</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSegmentEditorWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSegmentEditorWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>TimeSeriesAnnotation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TimeSeriesAnnotation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputSequenceSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TimeSeriesAnnotation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>segmentationSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TimeSeriesAnnotation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ultrasoundModelSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TimeSeriesAnnotation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>segmentEditorWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>448</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TimeSeriesAnnotation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>segmentationBrowserSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>572</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
